docs/rest-api.md

```diff
@@ -0,0 +1,181 @@
+# REST API Usage
+
+## Configuration
+
+Enable the rest API by adding the api_server section to your configuration and setting `api_server.enabled` to `true`.
+
+Sample configuration:
+
+``` json
+    "api_server": {
+        "enabled": true,
+        "listen_ip_address": "127.0.0.1",
+        "listen_port": 8080
+    },
+```
+
+!!! Danger: Security warning
+  By default, the configuration listens on localhost only (so it's not reachable from other systems). We strongly recommend to not expose this API to the internet, since others will be able to control your bot.
+
```

Comment:
"since others will be able to control your bot." => "since others will potentially be able to control your bot.